payloads/library/bunny_debug_helpers.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+################################################################################
+# Allow Debugging messages written to: "/root/udisk/debug/debug.txt"
+# on the BashBunny
+#
+# How this works?
+# 1) Once the lobrary is included in your payload, write text to
+#    the debug file with:
+#       echo "DEBUG MESSAGE" >> "${DEBUG_FILE}"
+#	    OR
+#	    debug_log "DEBUG MESSAGE"
+#	    (To write to log with timestamps)
+# 2) After attack, Text can be read at: "/root/udisk/debug/debug.txt"
+#    on the BashBunny
+# 3) To turn off debugging, pass the OFF comand when including the helper
+#       source bunny_debug_helpers.sh OFF
+################################################################################
+
+################################################################################
+# Start Debug
+################################################################################
+if [ "$1" = "OFF" ]; then
+    DEBUG_STATE="OFF"
+else
+    DEBUG_STATE="ON"
+fi
+
+timestamp () {
+    echo "$(date +"%Y-%m-%d_%H-%M-%S"):"
+}
+
+start_debug () {
+        DEBUG_FILE="./debug/debug.txt"
```

Comment:
Suggestion: You could make the file name dependent on the date so you get a new log file every time the script executes rather than using the same file. That way you could keep multiple log files and keep them sorted.